src/components/documentBuilder/edit/getElementEditSchemas.tsx

```diff
@@ -253,9 +253,11 @@ function getElementEditSchemas(
         getClassNameEdit(elementName, {
           label: "Label Style",
           uiSchema: {
-            textVariant: false,
-            backgroundColor: false,
-            borders: false,
+            options: {
```

Comment:
Sorry, if you're trying to follow the defined one in the type, this should be `"ui:options"`
```suggestion
            "ui:options": {
```

Reply:
Makes sense. I'll swap em